backend/src/api/transactions/transaction.model.js

```diff
@@ -27,3 +27,20 @@ const transactionSchema = new Schema(
 );
 
 module.exports = mongoose.model('Transaction', transactionSchema);
+
+class TransactionModel {
```

Comment:
We've already talked with Marina about schema.statics approach. Please follow one code-style